### ISSUE TITLE
Confused about the result

### ISSUE THREAD

**Issue by scottshufe** [GeorgeTzannetos/ml-leaks-pytorch#1]
Hello George, thanks for your code first. 
I have tried to run it, but I feel confused about the results. I finished training the target model and the shadow model with no problem. But when I start training the attack model, it seems that the training loss does not change with the epoch num increasing. And I cannot understand the target set accuracy calculation method in your code.
Could you please answer these questions for me? Thanks in advance and look forward to your reply.

**Comment by scottshufe** [comment]
Moreover, I find a bug in the code
Please see: https://github.com/GeorgeTzannetos/ml-leaks-pytorch/blob/6f1f10aa739fea454a5d49a5613e0eb461385d26/train_eval.py#L122
I think the statement `return running_loss` should be out of the for loop?

**Comment by GeorgeTzannetos** [comment]
Hi Scott, thank you for the raised issue, and sorry for the late reply, but I wasn't actively working on that. I have updated the repository now. If you still have any questions, feel free to ask, and I will try to get back to you.

**Comment by GeorgeTzannetos** [comment]
Hi, thank you for the raised issue, and sorry for the late reply, but I wasn't actively working on that. I have updated the repository now based on your comment. If you still have any questions, feel free to ask, and I will try to get back to you.
